components/ai_chat/resources/page/chat_ui.tsx

```diff
@@ -83,21 +86,35 @@ function App() {
 
   return (
     <AIChatContextProvider
-      selectedConversationUuid={selectedConversationUuid}
+      isDefaultConversation={!selectedConversationUuid}
       onNewConversation={handleNewConversation}
       onSelectConversationUuid={handleSelectConversationUuid}
     >
       {conversationAPI && (
         <ConversationContextProvider {...conversationAPI}>
           <BraveCoreThemeProvider>
-            <Main />
+            <Content />
           </BraveCoreThemeProvider>
         </ConversationContextProvider>
       )}
     </AIChatContextProvider>
   )
 }
 
+function Content () {
+  const aiChatContext = useAIChat()
+
+  if (aiChatContext.isStandalone === undefined) {
+    return <div>"loading..."</div>
+  }
+
+  if (!aiChatContext.isStandalone) {
```

Comment:
@petemill I wonder if this would be a better match for `loadTimeData` rather than an asynchronous API call, as it means we'll block loading this page every time until we've:
1. Loaded the page
2. Loaded/parsed/executed the JS
3. Done a React render
4. Done a mojom round trip to the browser to determine whether we're in `isStandalone` mode
5. Done a React rerender

Reply:
Yes as long as the timing works out for the compaison of the web_ui WebContents to the active WebContents to be done in the constructor.